src/server/dflycmd_test.cc

```diff
@@ -0,0 +1,200 @@
+// Copyright 2023, DragonflyDB authors.  All rights reserved.
+// See LICENSE for licensing terms.
+//
+
+#include <absl/flags/reflection.h>
+
+#include <string>
+#include <string_view>
+
+#include "base/gtest.h"
+#include "base/logging.h"
+#include "facade/facade_test.h"
+#include "server/test_utils.h"
+
+namespace dfly {
+namespace {
+using namespace std;
+
+class DflyFamilyTest : public BaseFamilyTest {
+ public:
+  DflyFamilyTest() {
+    auto* flag = absl::FindCommandLineFlag("cluster_mode");
```

Comment:
Because you are setting the cluster mode flag here, I would call this tests DflyClusterTest , cause we might want to add to this file more test which are not under cluster mode